actionpack/lib/action_controller/metal/helpers.rb

```diff
@@ -53,9 +53,8 @@ class << self; attr_accessor :helpers_path; end
     include AbstractController::Helpers
 
     included do
-      class_attribute :helpers_path, :include_all_helpers
-      self.helpers_path ||= []
```

Comment:
Probably shouldn't replace this with a default:, right? Given that it checks the parent class first.